core/src/mindustry/world/blocks/distribution/MassDriver.java

```diff
@@ -50,6 +51,14 @@ public MassDriver(String name){
         config(Integer.class, (MassDriverBuild tile, Integer point) -> tile.link = point);
     }
 
+    @Override
+    public void setStats(){
+        super.setStats();
+
+        stats.add(Stat.shootRange, range / tilesize, StatUnit.blocks);
+        stats.add(Stat.reload, 60f / (reloadTime + 1), StatUnit.none);
```

Comment:
The +1 shouldn't be necessary, that was a turret-specific quirk.

Reply:
Interesting. I wonder why.